Makefile

```
@@ -393,6 +393,7 @@ define VS_LAUNCH
             "request": "launch",
             "servertype": "stutil",
             "cwd": "$${workspaceRoot}",
+            "gdbPath": "${ARM_GCC_PATH}/arm-none-eabi-gcc",
```

E tava vendo [aqui](https://github.com/Marus/cortex-debug/blob/master/debug_attributes.md), acho que não precisa do caminho completo

```suggestion
            "gdbPath": "arm-none-eabi-gdb",
```